kernel/bpf/test_bpf/runtest.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Include rhts environment
+. /usr/bin/rhts-environment.sh
+. /usr/share/beakerlib/beakerlib.sh
+
+# Logic:
+# 1. Download the SRPM and install the SRPM
```

Comment:
Since this test expects existing SRPM, is it correct to assume that it's only applicable for Brew/koji and RPM builds? For upstream kernels (e.g stable and mainline) we only build tarballs, no RPMs.

Reply:
That also makes sense, but can you point me a way or library function to download the tarball? What I need is only the specific module source code so as long as I have the tarball I can change it.

Reply:
You can find some builds at https://artifacts.cki-project.org/builds/
They are installed using the kpkginstall task, specifically this function: https://github.com/CKI-project/tests-beaker/blob/master/distribution/kpkginstall/runtest.sh#L63

Reply:
Great! I'll take a look for this.

Reply:
Hi, I think this differs from what I need. I checked the builds and did not see any source included in the tarballs. Moreover the test_bpf is not pre-build. Given I need the source code, is there a library to do so?

Reply:
I don't think there's any tooling for that. We'd have to pass the repo URL to the task, including the URLs of the patches that are being tested and you'd have to apply them in here to reconstruct the "full" tree of tested kernel. It's possible but I don't think it's worth it and the test shouldn't be dealing with things like this, TBH. Thanks for looking into the option!

Reply:
Thanks for the explain.
So I think right now this case will only run against kernel that is delivered by RPM package.